Dockerrun.aws.json

```diff
@@ -1,7 +1,7 @@
 {
   "AWSEBDockerrunVersion": "1",
   "Image": {
-    "Name": "YOUR-DOCKER-HUB-ID/YOUR-IMAGE-NAME",
+    "Name": "kylety1er/saverlife-a",
```

Comment:
don't forget to go back in and un-hardcode your docker hub id. Other than that looks great!